### PR TITLE
Add build manifest for org.gnome.Polari

### PR DESCRIPTION
**Pull request by fmuellner**

### CODE REVIEW

File: org.gnome.Polari.json [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    "cleanup": ["/bin/m*","/bin/wraptool","/include","/lib/*.la",
+                "/lib/pkgconfig","/lib/polari/*.la","/lib/python*",
+                "/share/aclocal", "/share/autoconf-archive",
+                "/share/dbus-1/services/org.freedesktop*", "/share/gir-1.0",
+                "/share/gtk-doc", "/share/info", "/share/man",
+                "/share/polari", "/share/telepathy"
+    ],
+    "modules": [
+        {
+            "name": "autoconf-archive",
```
Comment (TingPing) [inline org.gnome.Polari.json]: Already in the Sdk.

**Comment by fmuellner** [comment]
Yay, one module less (used to not be the case: https://git.gnome.org/browse/polari/commit/?id=6d1d43640f2) ...

Branch updated.

**Comment by TingPing** [comment]
Test build: https://flathub.org/builds/#/builders/5/builds/213

**Comment by TingPing** [comment]
Looks like gjs crashes on aarch64. Dunno who wants to debug that or if we should just ignore that arch for now.

**Comment by fmuellner** [comment]
I can't find an existing bug report off-hand, but it's not a new problem - see https://gnome1.codethink.co.uk//logs/build-2017-05-03-130030/build-gnome-apps-nightly-master-org.gnome.Documents-aarch64.txt for a corresponding failure on the gnome-apps-nightly builder.

**Comment by alexlarsson** [comment]
Lets just disable aarch64 builds for now.

**Comment by ptomato** [comment]
Fedora has some aarch64 patches for mozjs38: http://pkgs.fedoraproject.org/cgit/rpms/mozjs38.git/commit/?id=3e8a25cad359f03d291c22e58b8b34c9a124ac11

**Comment by Barthalion** [comment]
@alexlarsson You will probably take care of it yourself, but any pull request submitted for Gnome Platform repo on Github is being automatically closed by some bot. Can you switch it off or patches should be sent to xdg-app mailing list?

**Comment by fmuellner** [comment]
I have now included mozjs38 in the modules and applied the fedora patch mentioned by @ptomato. This isn't something that I would really want in the Polari flatpak, but it's a good way to test whether the patch fixes the aarch64 issue, so we can add it to the gnome runtime.

**Comment by TingPing** [comment]
Build failed: https://flathub.org/builds/#/builders/5/builds/319

**Comment by fmuellner** [comment]
So it looks like the Fedora patch doesn't make a difference ...

**Comment by fmuellner** [comment]
I'm cheating a bit here, but I don't want to block on someone figuring out the aarch64 issue eventually (I don't even have a way to debug the issue, so I'm afraid I won't be that someone) ...

**Comment by fmuellner** [comment]
Ping?

**Comment by Barthalion** [comment]
You can just disable building for `aarch64` with the file like the one I used for PPSSPP [here](https://github.com/flathub/org.ppsspp.PPSSPP/blob/master/flathub.json). (Obviously you need to change the architecture.)

**Comment by fmuellner** [comment]
@Barthalion Thanks for the tip!

**Comment by TingPing** [comment]
Repository has been created: https://github.com/flathub/org.gnome.Polari

**Pull request closed without merging**